project/newrelic_webmon/newrelic.ini

```diff
@@ -0,0 +1,195 @@
+# ---------------------------------------------------------------------------
```

Comment:
What's the advantage of that? (I'd worry that we'd miss one of the multiple places we have to change when we add environment variables to our processes.)